packages/gatsby/src/utils/engines-helpers.ts

```diff
@@ -32,7 +32,7 @@ function getCDNObfuscatedPath(path: string): string {
   return `${store.getState().status.cdnObfuscatedPrefix}-${path}`
 }
 
-export const LmdbOnCdnPath = getCDNObfuscatedPath(`data.mdb`)
+export const getLmdbOnCdnPath = (): string => getCDNObfuscatedPath(`data.mdb`)
```

Comment:
making it lazy here, as `getCDNObfuscatedPath` is potentially using some values from cache that might not have been restored yet at import time (adapter restore it later)

that was causing a bug where generated path was different on each build while it should be the same to avoid publishing multiple versions of db to cdn